proxy/coco/src/state.rs

```diff
@@ -228,8 +228,10 @@ impl State {
     ///
     /// * if opening the storage fails
     pub fn list_owner_project_refs(&self, urn: &RadUrn) -> Result<Refs, Error> {
-        let storage = self.api.storage();
-        storage.rad_signed_refs(urn).map_err(Error::from)
+        let storage = self.api.storage().reopen()?;
```

Comment:
Do we know if we have to `reopen` here? :thinking:

Reply:
Leftovers.

Reply:
Thought as much :grin: